tests/integration-tests/tests/update/test_update.py

```diff
@@ -163,7 +163,13 @@ def test_update_slurm(region, pcluster_config_reader, s3_bucket_factory, cluster
                     "instances": [
                         {
                             "instance_type": "c5.xlarge",
-                        }
+                        },
```

Comment:
Should we update the `initial_queues_config` as well?

Reply:
you are right, I missed the initial config check